README.md

```diff
@@ -3,11 +3,9 @@
   <i>An open source project from Data to AI Lab at MIT.</i>
 </p>
 
-[![][pypi-img]][pypi-url] [![][travis-img]][travis-url]
-
 # Reversible Data Transforms
 
-This a python library used to transform data for data science libraries and preserve the transformations in order to reverse them as needed.
+[![][pypi-img]][pypi-url] [![][travis-img]][travis-url]
```

Comment:
Move this up, over the title, and remove the variables. Also add the downloads.

Take the [Copulas README](https://github.com/DAI-Lab/Copulas/blob/master/README.md) as an example.

**NOTE**: Skip codecov for now, this will be added later on a different PR.

```
[![PyPi Shield](https://img.shields.io/pypi/v/copulas.svg)](https://pypi.python.org/pypi/copulas)
[![Travis CI Shield](https://travis-ci.org/DAI-Lab/Copulas.svg?branch=master)](https://travis-ci.org/DAI-Lab/Copulas)
[![Coverage Status](https://codecov.io/gh/DAI-Lab/Copulas/branch/master/graph/badge.svg)](https://codecov.io/gh/DAI-Lab/Copulas)
[![Downloads](https://pepy.tech/badge/copulas)](https://pepy.tech/project/copulas)


# Copulas

* Free software: MIT license
* Documentation: https://DAI-Lab.github.io/Copulas
* Homepage: https://github.com/DAI-Lab/Copulas
```